config/core/configmaps/autoscaler.yaml

```diff
@@ -137,3 +137,11 @@ data:
     # that should be used if none is specified. If omitted, the Knative
     # Horizontal Pod Autoscaler (KPA) is used by default.
     pod-autoscaler-class: "kpa.autoscaling.knative.dev"
+
+    # The capacity of a single activator task.
+    # The `unit` is one concurrent request proxied by the activator.
+    # activator-capacity must be at least 1.
+    # This value is used for computation of the Activator subset size.
+    # See the algorithm here: http://bit.ly/38XiCZ3.
+    # TODO(vagababov): tune after actual benchmarking.
+    activator-capacity: "100"
```

Comment:
Perhaps use 100.0 to convey it's a float?

Reply:
Actually, is there a point in allowing fractional values here? 😜

Reply:
All other flags are floats and target is float and all the math is float64, so why not?

Reply:
Pushed .0.